proposals/accepted/047-standard-library-reform.rst

```diff
@@ -0,0 +1,255 @@
+Standard library reform: Split base
+===================================
+
+.. sectnum::
+.. contents::
+
+Abstract
+--------
+
+Issues with the standard library are holding back the Haskell ecosystem.
+The problems and solutions are multifaceted, and so the Haskell Foundation in its "umbrella organization" capacity is uniquely suited to coordinate the fixing of them.
```

Comment:
I'm not sure that this matters. The HF should do useful stuff that isn't uniquely useful as well - we should just argue that this makes things better.